LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018-present Jop de Klein
```

Comment:
Please revise this to `styled-components`

Reply:
```suggestion
Copyright (c) 2018-present styled-components
```

Reply:
Done 👍